### PR TITLE
[#8957] InstructorCourseDetailsPageUiTest.testRemindAction failing on live server

### LINKED ISSUE #8957: InstructorCourseDetailsPageUiTest.testRemindAction failing on live server
v6.7.0
Possibly a case of not using a real student account; error not caught by CI because email testing is omitted in dev server
>java.lang.AssertionError: 
	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
	at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)
	at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:33)
	at teammates.test.cases.BaseTestCase.assertTrue(BaseTestCase.java:127)
	at teammates.test.cases.browsertests.InstructorCourseDetailsPageUiTest.testRemindAction(InstructorCourseDetailsPageUiTest.java:183)
	at teammates.test.cases.browsertests.InstructorCourseDetailsPageUiTest.allTests(InstructorCourseDetailsPageUiTest.java:60)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:72)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:123)

### PR DESCRIPTION
**Pull request by xpdavid**
Fixes #8957

It doesn't make sense to use retry manger to check an email is **NOT** received. <strike>The only method I can think of is to wait for a reasonable amount of time and then check.</strike>

Updates:

Adopted both methods: waiting for a reasonable amount of time and the suggestion from @damithc .

### CODE REVIEW

**Comment by damithc** [comment]
> It doesn't make sense to use retry manger to check an email is **NOT** received. The only method I can think of is to wait for a reasonable amount of time and then check.

Perhaps send another email and ensure the second email is received but not the first? This assumes emails are delivered in order, which may not be the case.

**Comment by xpdavid** [comment]
@tshradheya a gentle reminder :)

**Comment by xpdavid** [comment]
@joanneong Shall we get this reviewed so it won't fail in V6.14.0?

File: src/test/java/teammates/test/cases/browsertests/InstructorCourseDetailsPageUiTest.java [hunk 1]
```diff
@@ -57,12 +64,86 @@ public void allTests() throws Exception {
...
+        }
+        detailsPage.waitForTextsForAllStatusMessagesToUserEquals(
+                Const.StatusMessages.COURSE_REMINDER_SENT_TO + student2.email);
+
+        // Hiding of the 'Send invite' link is already covered by content test.
+        //  (i.e., they contain cases of both hidden and visible 'Send invite' links.
+
+        ______TS("action: remind all");
+
+        detailsPage.clickRemindAllAndCancel();
```
Comment (joanneong) [inline src/test/java/teammates/test/cases/browsertests/InstructorCourseDetailsPageUiTest.java]: Missing check for whether student has received reminder after cancelling this action.

**Pull request merged**